packages/SwingSet/src/kernel/kernel.js

```diff
@@ -439,66 +439,6 @@ export default function buildKernel(
     return null;
   }
 
-  async function deliverToTarget(target, msg) {
-    insistMessage(msg);
-    /** @type { PolicyInput } */
-    let policyInput = ['none'];
-    const { type } = parseKernelSlot(target);
-    if (type === 'object') {
-      const vatID = kernelKeeper.ownerOfKernelObject(target);
-      if (vatID) {
-        policyInput = await deliverToVat(vatID, target, msg);
-      } else if (msg.result) {
-        resolveToError(msg.result, VAT_TERMINATION_ERROR);
-      }
-    } else if (type === 'promise') {
-      const kp = kernelKeeper.getKernelPromise(target);
-      if (kp.state === 'redirected') {
-        // await deliverToTarget(kp.redirectTarget, msg); // probably correct
-        // TODO unimplemented
-        throw new Error('not implemented yet');
```

Comment:
I assume we don't need to keep this?

Reply:
yeah, it was aspirational, but we'll figure something else out if/when we ever implement redirection